pkgs/stdenv/default.nix

```diff
@@ -38,28 +38,10 @@ let
 in
   if crossSystem != localSystem || crossOverlays != [] then stagesCross
   else if config ? replaceStdenv then stagesCustom
-  else { # switch
-    i686-linux = stagesLinux;
-    x86_64-linux = stagesLinux;
-    armv5tel-linux = stagesLinux;
-    armv6l-linux = stagesLinux;
-    armv6m-linux = stagesLinux;
-    armv7a-linux = stagesLinux;
-    armv7l-linux = stagesLinux;
-    armv7r-linux = stagesLinux;
-    armv7m-linux = stagesLinux;
-    armv8a-linux = stagesLinux;
-    armv8r-linux = stagesLinux;
-    armv8m-linux = stagesLinux;
-    aarch64-linux = stagesLinux;
-    mipsel-linux = stagesLinux;
-    mips64el-linux = stagesLinux;
-    powerpc-linux = /* stagesLinux */ stagesNative;
```

Comment:
You missed this one special case :)

Reply:
I mentioned it in the commit message. I don't think special case ought to exist, and I don't think anyone is using `powerpc-linux` as opposed to the newer Power arches anyways

Reply:
Sorry about that, makes sense.